services/repository/src/main/java/org/sagebionetworks/auth/services/AuthenticationServiceImpl.java

```diff
@@ -315,15 +199,24 @@ public void unbindExternalID(Long userId, OAuthProvider provider, String aliasNa
 	}
 
 	@Override
-	public LoginResponse login(LoginRequest request) throws NotFoundException {
+	public LoginResponse login(LoginRequest request) {
 		ValidateArgument.required(request, "request");
 		ValidateArgument.required(request.getUsername(), "LoginRequest.username");
 		ValidateArgument.required(request.getPassword(), "LoginRequest.password");
-		// Lookup the user.
-		PrincipalAlias pa = lookupUserForAuthentication(request.getUsername());
-		if(pa == null) throw new NotFoundException("Did not find a user with alias: "+request.getUsername());;
+		try {
+			// Lookup the user.
+			PrincipalAlias pa = userManager.lookupUserForAuthentication(request.getUsername());
+
+			// Fetch the user's session token
+			return authManager.login(pa.getPrincipalId(), request.getPassword(), request.getAuthenticationReceipt());
+		} catch (NotFoundException e) {
+			// see PLFM-3914
+			throw new UnauthenticatedException();
```

Comment:
pass cause

Reply:
I see no update to line 214.  There's only one commit in the PR.